### ISSUE TITLE
DK starting Area bugged quest

### ISSUE THREAD

**Issue by pe93** [azerothcore/azerothcore-wotlk#9175]
### Current Behaviour

When i try to finish the quest The Scarlet Apocalypse, i have to talk to the Lichking in Deaths Breach. Well he is not on his Platform. Heßs  in the Platform. You can still turn the quest in but its a little bit buggy.

### Expected Blizzlike Behaviour

He should Stand atop the platform.

### Steps to reproduce the problem

Start a Dk
make the storyquests till you get the quest The Scarlet Apocalypse
Try talking to the Lichking in Deats Breach.

### AC rev. hash/commit

b2f0893

### Operating system

Win 10 running with docker

**Comment by acidmanifesto** [comment]
This is current with 335 tc and 434 cpp. It has something about dealing with the game object or game object handling of that platform. something worth looking into later to see what the deal is with this as there is no issue of this on the master tc variations.

**Comment by acidmanifesto** [comment]
https://github.com/TrinityCore/TrinityCore/pull/22892

**Comment by Kitzunu** [comment]
https://github.com/azerothcore/azerothcore-wotlk/issues/5714

**Comment by acidmanifesto** [comment]
We got a pr waiting testing right now to address this issue.

**Comment by PerryCS** [comment]
I JUST finished the entire DK questline yesterday and it's true... he wasn't there but all I had to do was logout and back in and poof... he was standing there.  I fresh pulled the source from 5 days ago as I couldn't upgrade the 2+ year old one.  This new version of the code feels more realistic.  Last time 2 years ago I went through the DK starting area I had to cheat quite a bit due to high number of mobs.  But, this time (after the rewrite or code update last year I think it was)... things seems to be more fair.  Didn't have to cheat at all this time to get through the entire DK section.

**Comment by Razzdazz11** [comment]
@acidmanifesto @Nyeriah need to reopen, as of #2789d5b87784 lich king is still under the ramp in the DK chain

**Comment by acidmanifesto** [comment]
> @acidmanifesto @Nyeriah need to reopen, as of #5df73f6 lich king is still under the ramp in the DK chain

Uncertain. Currently we identify a recent issue from another pr that seem to unlink scripts from compiling.  A solution will come soon so we will wait on that first.

**Comment by Voxstrasza** [comment]
> > @acidmanifesto @Nyeriah need to reopen, as of #5df73f6 lich king is still under the ramp in the DK chain
> 
> Uncertain. Currently we identify a recent issue from another pr that seem to unlink scripts from compiling. A solution will come soon so we will wait on that first.

I took a look at the issue and the model was clipping into the platform yielding inconsistent results on whether or not the model would stay on the platform.  I created a PR to adjust the spawn position.  Please take a look.
